lib/stream.js

```diff
@@ -41,8 +41,8 @@ function duplexOnError(err) {
 /**
  * Wraps a `WebSocket` in a duplex stream.
  *
- * @param {WebSocket} ws The `WebSocket` to wrap
- * @param {Object} [options] The options for the `Duplex` constructor
+ * @param {import("./websocket")} ws The `WebSocket` to wrap
+ * @param {import("node:stream").DuplexOptions} [options] The options for the `Duplex` constructor
```

Comment:
I think this is TypeScript-specific.

Reply:
Strange. I thought that this was. normal in JSDoc, but I can remove this as well.

Reply:
For the `import("./websocket")` case we can do this:

```diff
diff --git a/lib/stream.js b/lib/stream.js
index 230734b..1eaf7b6 100644
--- a/lib/stream.js
+++ b/lib/stream.js
@@ -1,5 +1,8 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "^WebSocket$" }] */
+
 'use strict';
 
+const WebSocket = require('./websocket');
 const { Duplex } = require('stream');
 
 /**

```

Reply:
Added.